modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -340,12 +340,29 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		return
 
 	if(target.key && user != target)
-		if(iscyborg(target))
+		var/mob/living/mob_occupant = target //Get mob living
+		if (target.getorgan(/obj/item/organ/brain) ) //Target the Brain
+			if(ai_controller?.ai_status == AI_STATUS_ON) // Is the character empty / AI Controlled
+				for (mob_occupant) // Get Target Client
```

Comment:
I don't quite get what you're doing here and I don't think it'll work

Reply:
So i based this off of the Examine code, It checks the target for a brain, then checks if it has AI_Stats_On suggesting it is no longer player controlled. I removed the For Occupant and Var as they arent needed.